src/Block.php

```diff
@@ -64,17 +114,31 @@ public function getImageUrl() {
     return FALSE;
   }
 
+  /**
+   * HTML Renders the block.
```

Comment:
It renders the block into a the block datastructure (see `hook_block_view()`) Only the content-subarray is a renderable array.